fakefs.gemspec

```diff
@@ -9,14 +6,11 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Chris Wanstrath", "Scott Taylor", "Jeff Hodges", "Pat Nakajima", "Brian Donovan"]
   spec.email         = ["chris@ozmm.org"]
   spec.description   = %q{A fake filesystem. Use it in your tests.}
-  spec.summary       = %q{A fake filesystem. Use it in your tests.}
-  spec.homepage      = "http://github.com/defunkt/fakefs"
+  spec.summary       = spec.description
```

Comment:
without description `WARNING:  no description specified`
without summary `ERROR:  While executing gem ... (Gem::InvalidSpecificationException) missing value for attribute summary`

so no :)